tests/modules/dastool/scaffolds2bin/test.yml

```diff
@@ -0,0 +1,14 @@
+- name: dastool scaffolds2bin test_dastool_scaffolds2bin
+  command: nextflow run tests/modules/dastool/scaffolds2bin -entry test_dastool_scaffolds2bin -c tests/config/nextflow.config
+  tags:
+    - dastool/scaffolds2bin
+    - dastool
+  files:
+    - path: output/dastool/test.scaffolds2bin.tsv
+      md5sum: 6e46c0be14dded7cb13af38f54feea47
+    - path: output/metabat2/bins/test.1.fa.gz
+      md5sum: 2b297bf557cc3831b800348859331268
+    - path: output/metabat2/test.tsv.gz
+      md5sum: 619338fa5019e361d5545ce385a6961f
```

Comment:
```suggestion
```

Reply:
It is reproducible with Docker/Singularity, so I would rather keep the md5sum check, and let it fail for conda.